src/validations/user.validation.js

```diff
@@ -0,0 +1,40 @@
+import Joi from 'joi';
+import myCustomJoi from 'joi-phone-number';
```

Comment:
Instead of using joi-phone-number ,Joi accept validation by regular expression please read well the documentation.